packages/pwa-buildpack/lib/Utilities/__tests__/createFileHash.spec.js

```diff
@@ -0,0 +1,54 @@
+const path = require('path');
+const fs = require('fs');
+
+const createFileHash = require('../createFileHash');
+
+const DEFAULT_SAMPLE_FILE_PATH = path.join(
+    path.resolve(__dirname).split('/packages')[0],
+    'packages',
+    'pwa-buildpack',
+    'lib',
+    'Utilities',
+    '__tests__',
+    'sampleFile.txt'
+);
+
+test('Should return a promise that resolves to a string', async () => {
+    const returnValue = createFileHash(DEFAULT_SAMPLE_FILE_PATH);
+
+    expect(returnValue instanceof Promise).toBeTruthy();
```

Comment:
Jest's [toBeInstanceOf](https://jestjs.io/docs/en/expect#tobeinstanceofclass) assertion reads nicer here (it uses `instanceOf` underneath).